addon/doxmlparser/examples/doxy2plantuml/main.cpp

```diff
@@ -0,0 +1,222 @@
+
+#include <doxmlintf.h>
+
+#include <iostream>
+#include <vector>
+#include <string>
+#include <sstream>
+#include <fstream>
+#include <map>
+#include <filesystem>
+#include <stack>
+
+
+
+int main(int argc, char **argv) {
+	std::cout << "doxy2plantuml: started" << std::endl;
+
```

Comment:
This look to me more like a debug statement. For automatic processing I think this might be disturbing.

Reply:
Removed the prints in the latest commit